src/js/common/schemaform/definitions/dateRange.js

```diff
@@ -3,17 +3,23 @@ import * as date from './date';
 
 import commonDefinitions from 'vets-json-schema/dist/definitions.json';
 
-export const schema = commonDefinitions.dateRange;
+export const schema = {
+  type: 'object',
+  properties: {
+    from: commonDefinitions.date,
+    to: commonDefinitions.date
+  }
+};
```

Comment:
I mainly made this change because the common dateRange definition use `$ref`, which is JSON Schema's way to share common schema definitions within a schema. It makes it more difficult to use these common definitions in places because we have to handle including the referenced schema in places. So I've been trying to avoid it in our common definitions.